src/mumble/MainWindow.cpp

```diff
@@ -859,28 +859,7 @@ static void recreateServerHandler() {
 
 void MainWindow::openUrl(const QUrl &url) {
 	g.l->log(Log::Information, tr("Opening URL %1").arg(url.toString().toHtmlEscaped()));
-	if (url.scheme() == QLatin1String("file")) {
-		QFile f(url.toLocalFile());
-		if (! f.exists() || ! f.open(QIODevice::ReadOnly)) {
-			g.l->log(Log::Warning, tr("File does not exist"));
-			return;
-		}
-		f.close();
-
-		QSettings *qs = new QSettings(f.fileName(), QSettings::IniFormat);
-		qs->setIniCodec("UTF-8");
-		if (qs->status() != QSettings::NoError) {
-			g.l->log(Log::Warning, tr("File is not a configuration file."));
-		} else {
-			qSwap(qs, g.qs);
-			g.s.load();
-			qSwap(qs, g.qs);
 
-			g.l->log(Log::Warning, tr("Settings merged from file."));
-		}
-		delete qs;
-		return;
-	}
```

Comment:
This code was executed since the parameter for `-c path` was also a valid `file://` URL. So using the parameter would always trigger the merging of the `file` with itself.

It would probably be possible to prevent this while still keeping the config merging feature, but I think that it's not needed since it isn't documented and could cause trouble. If you want to run a heavily modified config and you accidently omit `-c` this would merge the modified file into the regular config, which could be very annoying.

Reply:
Agreed - that was a weird feature to begin with :sweat_smile: